### PR TITLE
Add documentation notes about compiling node_modules

### LINKED ISSUE #1100: Document how to Babel-transpile dependencies in node_modules too
Currently Neutrino intentionally does not compile `node_modules` by default, since packages should be compiling down to ES5 themselves (even when using the `module` property in `package.json`). There is an increasing (and unfortunate) trend within the react-**native** packages to not do this - which resulted in confusion in #1046. (See facebook/create-react-app#1125 and facebook/create-react-app#3776 for more history on this area)

I believe the Neutrino default is currently the right thing to do (particularly since unlike CRA, it's easy to override), however it would be good to document how a project can enable the transpiling of `node_modules` - either via a single section on `@neutrinojs/compile-loader` or else a mention on all of the top-level presets.

The new section should include something like:
https://github.com/neutrinojs/neutrino/issues/1046#issuecomment-416052505

### PR DESCRIPTION
**Pull request by eliperelman**
Fixes #1100.

### CODE REVIEW

File: packages/web/README.md [hunk 1]
```diff
@@ -497,6 +497,9 @@ By following the [customization guide](https://neutrinojs.org/customization/) an
 you can override and augment the build by by providing a function to your `.neutrinorc.js` use array. You can also
 make these changes from the Neutrino API in custom middleware.
 
+If the need arises, you can also compile `node_modules` by referring to the relevant
+[`compile-loader` documentation](https://neutrinojs.org/packages/compile-loader/#compiling-node_modules).
```
Comment (edmorley) [inline packages/web/README.md]: For the other presets, this is under the `Customizing` section. Should this be too? (Happy either way)

Comment (eliperelman) [reply packages/web/README.md]: Good idea, I'll move it.

**Comment by ackvf** [comment]
The link is dead now, is it possible to find the relevant documentation anywhere?

**Comment by edmorley** [comment]
For the in-release-candidate Neutrino 9 (`master` branch), it's here:
https://master.neutrinojs.org/packages/compile-loader/#compiling-node_modules

To try out v9, see #1129.

**Pull request merged**